src/topology/algebra/group.lean

```diff
@@ -493,4 +493,20 @@ lemma nhds_is_mul_hom : is_mul_hom (λx:α, 𝓝 x) := ⟨λ_ _, nhds_mul _ _⟩
 
 end
 
+@[to_additive]
+lemma tendsto_inv_nhds {α : Type*} [group α]
+  [topological_space α] [topological_group α] {a : α} :
+  tendsto has_inv.inv (nhds a) (nhds (a⁻¹)) :=
```

Comment:
Isn't it `continuous_inv.tendsto a`?

Reply:
You should not mark a conversation as resolved if you have not resolved it :) The comment by Yury is saying that this lemma has a one-line proof, namely, what he says.

Reply:
Also, this statement should probably be next to `continuous_inv` in the very same file.

Reply:
Oh sorry, my mind just reduced his comment to "you shouldn't make this lemma", so when you agreed to put it anyway I just put it back without thinking about the proof... I change this right now